.travis.yml

```diff
@@ -1,9 +1,15 @@
+sudo: false
 language: node_js
 node_js:
-  - '0.11'
+  - 4.4
```

Comment:
@conf I know, but the code doesn't touch that could be broken on Node.js upgrade, so I don't see a point.